projects/rocprofiler-sdk/samples/openmp_target/CMakeLists.txt

```diff
@@ -30,8 +30,7 @@ target_link_libraries(
     PRIVATE rocprofiler-sdk::rocprofiler-sdk rocprofiler-sdk::samples-build-flags
             rocprofiler-sdk::samples-common-library)
 
-set(DEFAULT_GPU_TARGETS "gfx906" "gfx908" "gfx90a" "gfx942" "gfx950" "gfx1100" "gfx1101"
-                        "gfx1102")
+set(DEFAULT_GPU_TARGETS ${ROCPROFILER_GPU_TARGETS})
```

Comment:
This is invalid when samples are build against an install -- `ROCPROFILER_GPU_TARGETS` is not set. Furthermore, all of these targets may not be supported by OpenMP